.travis.yml

```diff
@@ -1,39 +0,0 @@
-dist: bionic
```

Comment:
To assist with merging this, could you please not delete .travis.yml within this pull request but instead leave it for a later one?

That means we'll be able to do:

- submit this change which activates github-actions
- actions & travis will both temporarily be enabled
- review the behavior of the actions, confirm they're working properly
- do any fixes to actions if needed
- then finally delete travis

If we delete the travis config immediately I'd worry that we'll find out after submitting that the actions don't work, so the repo would have no working CI for a while.